src/index.scss

```diff
@@ -0,0 +1,38 @@
+body {
```

Comment:
The `global.scss` serves the purpose that the `index.scss` has, so let's...
1. Move all the `@font-face` to `global.scss`
2. Delete `index.scss` 

On that note, can we also move all the fonts you downloaded (the `.tff` files) into the `assets/` folder? Under a `font` directory? I think that'd make more sense, lemme know when you think